.github/workflows/publish.yml

```diff
@@ -34,19 +29,12 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         with:
-          python-version: |
-            3.7
-            3.8
-            3.9
-            3.10
-            3.11
-            3.12
-            3.13
+          python-version: 3.13
       - name: Build wheels
         uses: PyO3/maturin-action@v1
         with:
           target: ${{ matrix.platform.target }}
-          args: --release --out dist -i python3.7 python3.8 python3.9 python3.10 python3.11 python3.12 python3.13 pypy3.7 pypy3.8 pypy3.9 pypy3.10
+          args: --release --out dist --find-interpreter
```

Comment:
Most likely, find-interpreter now does nothing, but leaving this in to see if it will pick up PyPy automatically.